### PR TITLE
Issue 3: Image Index

### PR DESCRIPTION
**Pull request by gcpreston**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  def home; end
+  def home
+    @images = Image.all.order(created_at: :desc)
```
Comment (kkirasich) [inline app/controllers/application_controller.rb:5]: Probably need a test to make sure the ordering is correct

File: test/controllers/application_controller_test.rb [hunk 2]
```diff
@@ -6,4 +6,14 @@ class ApplicationControllerTest < ActionDispatch::IntegrationTest
...
     assert_select 'h1', 'Welcome to Ropes!'
   end
+
+  test 'image index descending order' do
+    img = Image.create(link: 'https://example.com')
+
+    get root_path
+    assert_select 'img' do |images|
+      assert_equal images.length, Image.count
+      assert_equal images[0]['src'], img.link
```
Comment (kkirasich) [inline test/controllers/application_controller_test.rb:16]: For order we'll need at least an `img1` and `img2`

**Review by kkirasich — APPROVED** [review]
:shipit:

**Comment by gcpreston** [comment]
[Issue 3: Image Index](https://trello.com/c/4IFpoHT2/381-issue-3-image-index)

**Pull request merged**